Sources/Controllers/2.0/BottomSheet/OABaseBottomSheetViewController.h

```diff
@@ -22,6 +22,7 @@
 
 @property (nonatomic, readonly) CGFloat initialHeight;
 
+- (CGFloat) getHeaderViewHeight;
```

Comment:
Seems redundant. If you want to adjust the initial height of the BottomSheet, simply override the getInitialHeight method in the subclass. Also, note that the ```self.headerView.frame``` is aavailable in the subclass as well.